tests/test_sessions.py

```diff
@@ -476,6 +476,86 @@ def test_session_by_id_links(client):
     )
     assert rv.status_code == 200
 
+def test_session_by_id_router_interface_add_delete(client):
```

Comment:
This is a bit paranoid of me, but mininet is sometimes very finicky. Have you considered testing the following?
- Adding two of the same interface
- Adding two separate interfaces and then deleting one (or any other layered interaction
- Adding many, many interfaces (I almost want to see where it breaks and test up to it)

Reply:
Definitely good tests to add; like discussed I'll only test up to 25 interfaces added due to test suite runtime.

Reply:
sounds great, thanks